simde/x86/sse2.h

```diff
@@ -3694,7 +3701,9 @@ simde_mm_max_pd (simde__m128d a, simde__m128d b) {
       a_ = simde__m128d_to_private(a),
       b_ = simde__m128d_to_private(b);
 
-    #if defined(SIMDE_POWER_ALTIVEC_P7_NATIVE)
```

Comment:
Same here.

I'm guessing you were trying to preserve the order from other functions, but you don't need to worry about that.  The only reason NEON is usually first is because that's usually the first implementation we add.  In this case it just so happens that we added AltiVec first.

Reply:
> I'm guessing you were trying to preserve the order from other functions, but you don't need to worry about that. The only reason NEON is usually first is because that's usually the first implementation we add. In this case it just so happens that we added AltiVec first.

Yes, I was trying to preserve the order so that it is consistent with all the functions.

Reply:
Sorry, I didn't realize I have to fix this, I have fixed it now.
Thank You